src/domain/Avatars/AvatarEntity/AvatarEntity.examples.md

```diff
@@ -4,166 +4,113 @@
 import { Props } from '@Common';
 import { AvatarStatusDotColor } from '@Domain/Avatars/Avatar';
 
+<>
   <AvatarEntity
     initials='JW'
     statusDot={AvatarStatusDotColor.Indigo}
     primaryText='John Wick'
   />
-```
-
-```jsx
-import { Props } from '@Common';
-import { AvatarStatusDotColor } from '@Domain/Avatars/Avatar';
-
   <AvatarEntity
     initials='JW'
     statusDot={AvatarStatusDotColor.Indigo}
     primaryText='John Wick'
     secondaryText='Guy Killer'
   />
-```
-
-```jsx
-import { Props, Variables } from '@Common';
-import { AvatarStatusDotColor } from '@Domain/Avatars/Avatar';
-import { FontAwesomeIcon } from '@Domain/Icons';
-
   <AvatarEntity
     initials='JW'
     statusDot={AvatarStatusDotColor.Indigo}
-    statusIcon={
-      <FontAwesomeIcon type='solid' icon='arrow-right' color={Variables.Color.i400} size='small'/>
-    }
     primaryText='John Wick'
     secondaryText='Guy Killer'
     tertiaryText='Job Ending in 3 days'
   />
+</>
 ```
 
-#### Compact
+#### Small AvatarEntity
 
 ```jsx
 import { Props } from '@Common';
 import { AvatarStatusDotColor } from '@Domain/Avatars/Avatar';
 
+<>
   <AvatarEntity
+    size='small'
     initials='JW'
-    statusDot={AvatarStatusDotColor.Green}
+    statusDot={AvatarStatusDotColor.Indigo}
     primaryText='John Wick'
-    isCompact
   />
-```
-
-
-```jsx
-import { Props, Variables } from '@Common';
-import { AvatarStatusDotColor } from '@Domain/Avatars/Avatar';
-import { FontAwesomeIcon } from '@Domain/Icons';
-
   <AvatarEntity
+    size='small'
+    initials='JW'
+    statusDot={AvatarStatusDotColor.Indigo}
+    primaryText='John Wick'
+    secondaryText='Guy Killer'
+  />
+  <AvatarEntity
+    size='small'
     initials='JW'
-    statusDot={AvatarStatusDotColor.Green}
-    statusIcon={
-      <FontAwesomeIcon type='solid' icon='arrow-right' color={Variables.Color.i400} size='xsmall' />
-    }
+    statusDot={AvatarStatusDotColor.Indigo}
     primaryText='John Wick'
     secondaryText='Guy Killer'
-    isCompact
+    tertiaryText='Job Ending in 3 days'
   />
+</>
 ```
 
-#### isHoverable
+#### Compact AvatarEntity
+
+Compact style will hide the tertiary text and put primary text and secondary text on one line
```

Comment:
I don't understand the point of having compact and size as separate concepts... shouldn't compact be another type of size that can be specified?

Reply:
Design team are more keen to drop use case of compact in the future. The small type will change the size but keep the avatar label in two lines which is a different concept of compact to have 1 line text label.